src/plugins/intel_cpu/src/compiled_model.cpp

```diff
@@ -82,6 +86,13 @@ CompiledModel::CompiledModel(const std::shared_ptr<ov::Model>& model,
         do {
             for (auto&& task : tasks) {
                 task = [this] {
+#if defined(OV_CPU_WITH_ACL)
+                    std::shared_ptr<arm_compute::IScheduler> acl_scheduler = std::make_shared<ACLScheduler>();
```

Comment:
It doesn't make sense to instance a new `acl_scheduler` every call of this code, while it's used only once. Please move this line inside the lambda passed to the `std::call_once` routine. Also it will allow making lambda L92-93 capture list empty.

Reply:
@maxnick corrected